src/api/todos.ts

```diff
@@ -17,4 +17,10 @@ export default [
     completed: false,
     userId: 4,
   },
+  {
+    id: 3,
+    title: 'quis ut nam facilis et officia qui',
+    completed: false,
+    userId: 3,
+  },
 ];
```

Comment:
Consider adding a trailing comma after the last item in the array for better version control and readability. 📝